src/app/pages/TopicPage/Curation/CurationGrid/index.jsx

```diff
@@ -88,10 +92,12 @@ const CurationGrid = ({ promos }) => {
 
 CurationGrid.propTypes = {
   promos: arrayOf(shape({})),
+  headingStyle: string,
 };
 
 CurationGrid.defaultProps = {
   promos: [],
+  headingStyle: 'h2',
```

Comment:
I'd maybe go with `headingLevel: 2,` here since `style` could be confused with CSS. Also, it prevents from passing in `p` and making it a paragraph.

Reply:
Just changed this now if you want to have a look :)